tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-django{111,20,21,22}
+envlist = py35-django{22,30}
```

Comment:
we should add python 3.8 here (I guess you made this change manually)

Reply:
You are right. I've added it.